sonic_platform_base/sonic_eeprom/eeprom_tlvinfo.py

```diff
@@ -359,9 +359,9 @@ def update_eeprom_db(self, e):
             if not self.is_valid_tlvinfo_header(e):
                 print("EEPROM does not contain data in a valid TlvInfo format.")
                 return -1
-            total_len = (ord(e[9]) << 8) | ord(e[10])
+            total_len = (e[9] << 8) | e[10]
             fvs['Id String'] = e[0:7]
-            fvs['Version'] = ord(e[8])
+            fvs['Version'] = e[8]
             fvs['Total Length'] = total_len
             client.hmset("EEPROM_INFO|TlvHeader", fvs)
```

Comment:
When I testing I see syseepromd failed to start due to some error here, did you see this?

```
Traceback (most recent call last):
  File "/usr/local/bin/syseepromd", line 170, in <module>
    main()
  File "/usr/local/bin/syseepromd", line 167, in main
    syseepromd.run()
  File "/usr/local/bin/syseepromd", line 137, in run
    rc = self.post_eeprom_to_db()
  File "/usr/local/bin/syseepromd", line 75, in post_eeprom_to_db
    err = self._wrapper_update_eeprom_db(eeprom)
  File "/usr/local/bin/syseepromd", line 63, in _wrapper_update_eeprom_db
    return self.eeprom.update_eeprom_db(eeprom)
  File "/usr/local/lib/python2.7/dist-packages/sonic_platform_base/sonic_eeprom/eeprom_tlvinfo.py", line 366, in update_eeprom_db
    client.hmset("EEPROM_INFO|TlvHeader", fvs)
  File "/usr/local/lib/python2.7/dist-packages/redis/client.py", line 3075, in hmset
    return self.execute_command('HMSET', name, *items)
  File "/usr/local/lib/python2.7/dist-packages/redis/client.py", line 900, in execute_command
    conn.send_command(*args)
  File "/usr/local/lib/python2.7/dist-packages/redis/connection.py", line 725, in send_command
    self.send_packed_command(self.pack_command(*args),
  File "/usr/local/lib/python2.7/dist-packages/redis/connection.py", line 775, in pack_command
    for arg in imap(self.encoder.encode, args):
  File "/usr/local/lib/python2.7/dist-packages/redis/connection.py", line 120, in encode
    "bytes, string, int or float first." % typename)
redis.exceptions.DataError: Invalid input of type: 'bytearray'. Convert to a bytes, string, int or float first.
```

Reply:
Thanks. Looks like I missed a line. I have pushed a new commit. Can you please test the latest patch?

Reply:
looks good, syseepromd works well.